src/devices/Ak8963/README.md

```diff
@@ -0,0 +1,115 @@
+# AK8963 - Magnetometer
+
+The AK8963 is a magnetometer that can be controlled either thru I2C either thru SPI. It is present in other sensors like the [MPU9250](../Mpu9250/README.md). This implementation fully supports the I2C mode and the usage thru the MPU9250. It does not support SPI.
+
+## Usage
+
+You can find an example in the [sample](./samples/ak8963.sample.cs) directory. Usage is straight forward including the possibility to have a calibration.
+
+```csharp
+// Create an I2C setting
+I2cConnectionSettings mpui2CConnectionSettingmpus = new I2cConnectionSettings(1, Ak8963.Ak8963.DefaultI2cAddress);
+// This will use the default I2C interface
+Ak8963 ak8963 = new Ak8963(new UnixI2cDevice(mpui2CConnectionSettingmpus));
```

Comment:
Use new I2cDevice.Create instead if package available.

Reply:
should be available already